src/jobs/deploy_to_selfhosted.yml

```diff
@@ -11,7 +17,7 @@ steps:
   - deploy:
       name: Poll for installation availability
       command: |
-        helm poll -r codacy
+        helm poll -r codacy-<< parameters.environment >> -n codacy-<< parameters.environment >>
```

Comment:
I think you should receive the entire `release_name` and  `namespace` as a parameter.